src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PathologyRecordMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.PathologyRecord;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = @Autowired)
+public class PathologyRecordMapper {
+
+    private final PractitionerMapper practitionerMapper;
+
+    public PathologyRecord mapToPathologyRecord(final Message message) {
+        return PathologyRecord.builder()
+                .requester(practitionerMapper.mapRequester(message).orElse(null))
```

Comment:
Rather than set `requester` to null if it doesn't exist, we could set the `requester` **only** if present (since it's optional after all), for example:

```
        final PathologyRecord.PathologyRecordBuilder pathologyRecordBuilder = PathologyRecord.builder();
        
        practitionerMapper.mapRequester(message).ifPresent(pathologyRecordBuilder::requester);
        
        return pathologyRecordBuilder.build();
```

Reply:
I was thinking of doing something like that before, cheers 😄